vue/components/ui/atoms/button-icon/button-icon.vue

```diff
@@ -254,6 +258,18 @@ export const ButtonIcon = {
             };
             if (this.color) base["button-icon-" + this.color] = this.color;
             return base;
+        },
+        loaderStyle() {
+            const scaledSize = (this.size * 10) / 30;
+            const scaledBorderWidth = (this.size * 2) / 30;
+            const marginTop = (this.size * 2) / 30; // Fixes the "ball-clip-rotate" animation not being centered

```

Comment:
Where did you get these numbers from?

Reply:
I took the default values in storybook and used it as reference to maintain the same appearance.

30 is the size used in storybook, and 10 is the value used in
```
.button-icon .loader ::v-deep div {
    border-color: #848484 #848484 transparent #848484;
    height: 10px;
    width: 10px;
}
```

Reply:
Just divide per 15 then